### ISSUE TITLE
[Image] Error: invalid context 0x0

### ISSUE THREAD

**Issue by PhilippKrone** [facebook/react-native#1534]
Hello,

since upgrading to 0.5.0, I get a lot of the following errors, although everything seems to work as expected:

````javascript
 <Error>: CGContextTranslateCTM: invalid context 0x0. This is a serious error. This application, or a library it uses, is using an invalid context  and is thereby contributing to an overall degradation of system stability and reliability. This notice is a courtesy: please fix this problem. It will become a fatal error in an upcoming update.
<Error>: CGContextScaleCTM: invalid context 0x0. This is a serious error. This application, or a library it uses, is using an invalid context  and is thereby contributing to an overall degradation of system stability and reliability. This notice is a courtesy: please fix this problem. It will become a fatal error in an upcoming update.
<Error>: CGContextDrawImage: invalid context 0x0. This is a serious error. This application, or a library it uses, is using an invalid context  and is thereby contributing to an overall degradation of system stability and reliability. This notice is a courtesy: please fix this problem. It will become a fatal error in an upcoming update.
 <Error>: CGContextRestoreGState: invalid context 0x0. This is a serious error. This application, or a library it uses, is using an invalid context  and is thereby contributing to an overall degradation of system stability and reliability. This notice is a courtesy: please fix this problem. It will become a fatal error in an upcoming update.
````

Do you have any idea where this might come from? Else I'd start to disable one plugin after the other and see what might be causing this.

Thanks
Philipp

**Comment by PhilippKrone** [comment]
Small update: it seems to happen whenever I navigator.push() away from a certain (quite complex) screen. However, it does not happen when i push() towards this screen.

**Comment by rxb** [comment]
Also seeing this. Happens when I push away from a screen that has an image with just flex: 1 as width, rather than an explicit width.

**Comment by sahrens** [comment]
Nick is oncall this week and I have no idea what this error is.

**Comment by brentvatne** [comment]
Fair @sahrens :smile: - I don't have a clue either, we can wait for Nick then :+1:

**Comment by grabbou** [comment]
@nicklockwood Nick, have you had any luck with sorting that out? Not sure if I should debug these warnings and get them fixed by changing the styles or wait as the UI looks perfectly anyway.

**Comment by yusefnapora** [comment]
Hey guys, I just made a PR for this: #2278 - it just adds a check to the `RCTClipRect` function to catch destination sizes where width or height are zero and calculates them based on the aspect ratio of the source image.  Makes the scary warnings go away :)

**Comment by ide** [comment]
Merged in with 0.10.0-rc.
